sdk/openai/Azure.AI.OpenAI/tests/AudioSpeechTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.OpenAI.Tests;
+
+public class AudioSpeechTests : OpenAITestBase
+{
+    public AudioSpeechTests(bool isAsync)
+        : base(Scenario.AudioSpeech, isAsync) // , RecordedTestMode.Live)
+    {
+    }
+
+    [RecordedTest]
+    [TestCase(Service.Azure)]
+    [TestCase(Service.NonAzure)]
+    public async Task GetAudioSpeech(Service serviceTarget)
+    {
+        OpenAIClient client = GetTestClient(serviceTarget);
+        string deploymentOrModelName = GetDeploymentOrModelName(serviceTarget);
+
+        var requestOptions = new AudioSpeechOptions(
+            deploymentOrModelName, "Hello World", AudioSpeechVoice.Alloy
```

Comment:
It'd better to include optional parameters like format and speed as well.

AudioSpeechOptions audioSpeechOptions = new()
{
    DeploymentName = deploymentOrModelName,
    Input = "Hello World",
    Voice = AudioSpeechVoice.Nova,
    ResponseFormat = AudioSpeechOutputFormat.Mp3,
    Speed = 0.8f
};

Reply:
Resolved by https://github.com/ninghu/azure-sdk-for-net/pull/7/commits/77d060c02b7ff8ecb721d88f2ec7045df4a9a1c9